packages/beacon-node/src/api/rest/base.ts

```diff
@@ -43,7 +43,15 @@ export class RestApiServer {
     const server = fastify({
       logger: false,
       ajv: {customOptions: {coerceTypes: "array"}},
-      querystringParser: querystring.parse,
+      querystringParser: (str) =>
+        qs.parse(str, {
+          // defaults to 20 but Beacon API spec allows max items of 30
+          arrayLimit: 30,
+          // array as comma-separated values must be supported to be OpenAPI spec compliant
+          comma: true,
```

Comment:
koa openapi req/res validator uses this library as well and sets [`comma: true`](https://github.com/atlassian/koa-oas3/blob/aae8267a0b7f1cc9e436c449cbc168394ccc85d6/src/config.ts#L92), this is required to be OpenAPI spec compliant, see [Parameter Serialization](https://swagger.io/docs/specification/serialization/)